src/azure-cli/azure/cli/command_modules/network/_params.py

```diff
@@ -857,6 +856,7 @@ def load_arguments(self, _):
         c.argument('macsec_ckn_secret_identifier',
                    help='The connectivity key name (CKN) that stored in the KeyVault.')
         c.argument('macsec_cipher', arg_type=express_route_link_macsec_cipher_type, help='Cipher Method')
+        c.argument('macsec_sci_state', arg_type=get_three_state_flag(positive_label='Enabled', negative_label='Disabled', return_label=True), help='Sci mode', min_api='2020-06-01')
```

Comment:
Is this type the same as `admin_state` above? It seems very similar and is it possible that we can share a common arg_type?